in_toto/verifylib.py

```diff
@@ -219,11 +223,12 @@ def run_all_inspections(layout):
 
     inspection_links_dict[inspection.name] = link
 
+    # If the client requests persistent inspection links,
     # Dump the inspection link file for auditing
     # Keep in mind that this pollutes the verifier's (client's) filesystem.
-    filename = in_toto.models.link.FILENAME_FORMAT_SHORT.format(
-        step_name=inspection.name)
-    link.dump(filename)
+    if persist_inspection_links:
+      filename = FILENAME_FORMAT_SHORT.format(step_name=inspection.name)
```

Comment:
The direct import of `FILENAME_FORMAT_SHORT` was removed in a recent PR (#379) please use the full path.